cumulusci.yml

```diff
@@ -482,8 +483,6 @@ flows:
                 flow: deploy_unmanaged
             3:
                 flow: config_dev
-            4:
```

Comment:
@davidmreed This flow was causing a recursion issue (Infinite flows detected with flow config_unmanaged).  We discovered that the config_dev flow executes the config_unmanaged flow so removing the duplicate task resolved the issue, but we're wondering if you could help shed some light on why we were seeing the recursion error?

Reply:
CumulusCI's recursion checker is not particularly clever: it thinks recursion is taking place any time the same flow gets called twice during an overall execution. I'll make sure we have a WI to fix and/or clarify that messaging.